Hao add home country and remove geographical location on profile

zhongh opened the pull request.

Commit:
Update to individual--foaf-person.ftl

 to display nationality as last item under personal picture

Commit:
Update to individual--foaf-person.ftl

obtain variable homeCountry

zhongh committed:
Removed Geographic Focus from the top of a person's profile

mrpatrickwest commented:
Looks good to me.

@zednis ?

zednis commented:
Are we not adding the home country to the user profile?  All updates to the individual FTL are commented out and will have no affect.

zednis commented on themes/dco/templates/individual--foaf-person.ftl:
```diff
@@ -112,7 +115,7 @@
 
         <!-- Geographic Focus -->
-        <#include "individual-geographicFocus.ftl">
+        <!-- #include "individual-geographicFocus.ftl" -->
```
What if we deleted this include statement rather than just commenting it out?

mrpatrickwest replied:
I agree. Let's go ahead and just remove the comment and the commented out include.

zhongh replied:
done.

mrpatrickwest replied:
Did you push your change? Looks like it's still there.

zednis commented on src/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualResponseBuilder.java:
```diff
@@ -91,6 +91,14 @@ ResponseValues assembleResponse() throws TemplateModelException {
+		}
+		*/
+		
+		/*
```
Why is this addition commented out?

mrpatrickwest replied:
We were going to add home country to the top of the profile, but it's already at the top of the Overview section so I decided we didn't need it. But, just in case engagement doesn't agree, for now we can leave it commented out.

zednis commented on themes/dco/templates/individual--foaf-person.ftl:
```diff
@@ -49,6 +49,9 @@
         <#include "individual-webpage.ftl">
+        
+        <!-- Nationality -->
+        <!-- Nationality: ${homeCountry!}  -->
```
What are we trying to do here?  This is commented out and will have no affect.

mrpatrickwest replied:
Same comment as above. Just in case engagement comes back and wants us to add the home country to the top of the profile, leave it commented out for now.

zednis replied:
Then we should probably comment out both with <#-- stuff commented out --> so the FTL template engine does not attempt to resolve ${homeCountry!}

XML/HTML comment: <!-- not shown in browser -->
FTL comment: <#-- not processed by FTL engine -->

zhongh replied:
want me to uncomment or you are already doing it?

zednis replied:
@zhonghao1014 you can make the update

zhonghao1014 commented:
want me to uncomment or you are already doing it?

Commit:
show nationality

show nationality as last item under the picture in personal profile

Commit:
generating homeCountry variable

Commit:
Nationality commented out for now

zhongh commented:
So the nationality was commented out in the ftl file in ftl-style comment <#-- -->

Commit:
Commented out java code to grab the homeCountry information. If we need
it then we can uncomment it. FTL comments for displaying the
Nationality. Don't want it showing up in the output at all. Removed
Geographic focus all together.

mrpatrickwest commented:
@zednis @zhonghao1014 are we good with this and ready to merge?

zhongh commented:
I am good.

zednis commented:
:shipit:
